ap-datasets.adoc

```diff
@@ -0,0 +1,414 @@
+a| Parameters added to terrestrial monitoring through Arctic PASSION funding include:
+
+- Permafrost/Ground temperature
+
+|In Work 
+|Otso Suominen 
+|INPO - INTERACT
+|  
+|  
+|Work is in progress to integrate data directly through GEM using schema.org served through INTERACT 
+
+|CNR Arctic Station “Dirigibile Italia”, Svalbard, Norway 
+a| Parameters added to terrestrial monitoring through Arctic PASSION funding include:
+
+- Soil moisture
+- Air temperature
+- Air humidity
+- Permafrost/Ground temperature
+- Snow depth
+|In Work 
+|Mauro Mazzola        
+|INPO - INTERACT 
+|https://iadc.cnr.it/cnr/[CNR/IADC]
+|  
+|Data from IADC are actively harvested into the catalogue.
+
+Need to check if data are tagged with Arctic PASSION.
+
+|Terrestrial climate and cryosphere data from Rif field station, Iceland
+a| Parameters added to terrestrial monitoring through Arctic PASSION funding include:
+
+- Permafrost/Ground temperature
+- Snow depth
+- Net radiation
+- Short wave incoming and outgoing radiation
+- Long wave incoming and outgoing radiation
+- Soil heat flux
+|In Work 
+|Pedro Rodrigues 
+|INPO - INTERACT
+|  
+|  
+| 
+
+|Terrestrial climate and cryosphere data from NIBIO Svanhovd research station, Norway 
+a| Parameters added to terrestrial monitoring through Arctic PASSION funding include:
+
+- Permafrost/Ground temperature
+- Incoming and outgoing short wave radiation
+- Incoming and outgoing long wave radiation
+- Net radiation
+- Heat flux
+|In Work 
+|Cornelya Klutsch 
+|INPO - INTERACT – International Network for Terrestrial Research and Monitoring in the Arctic - INPO (SE)
+|  
+|  
+|Data are currently not tagged as Arctic PASSION, some of these are being processed in MET/ADC. 
+
+|Terrestrial climate and cryosphere data from Churchill Northern Studies Centre, Canada 
+a|Parameters added to terrestrial monitoring through Arctic PASSION
+funding include:
+
+- Air temperature
+- Air humidity
+- Short wave incoming and outgoing radiation
+- Long wave incoming and outgoing radiation
+- Wind speed
+- Wind direction
+- Air pressure
+|In Work 
+|Joni Reimer 
+|INPO - INTERACT
+|  
+|  
+| 
+
+////
+Double check that this is not a duplicate of the Bamber dataset listed above.
+////
+|Calving Front Dataset for Marine-Terminating Glaciers in Svalbard 1985-2023
+a|This is a new high-resolution calving front dataset of 149 marine-terminating glaciers in Svalbard, comprising 124919 glacier calving front positions during the period of 1985-2023. This dataset was generated using a novel automated deep learning framework and multiple optical and SAR satellite images from Landsat, Terra-ASTER, Sentinel-2, and Sentinel-1 satellite missions. 
+|Complete
+|Tian Li
+|University of Bristol
+|AWI/PANGAEA
+|
+|
+|This dataset was published in https://zenodo.org/records/10407266[Zenodo] and will need to be integrated in Arctic PASSION data holdings through filling in manually the web form for metadata. Zenodo is not recommended for data publishing.
+
```

Comment:
This is already integrated in the portal. If you want to add the publication date:  2023-12-18

Reply:
Thanks I will update the record.